core/client/src/light/fetcher.rs

```diff
@@ -432,6 +446,10 @@ pub mod tests {
 		fn remote_changes(&self, _request: RemoteChangesRequest<Header>) -> Self::RemoteChangesResult {
 			err("Not implemented on test node".into())
 		}
+
+		fn remote_body(&self, _request: RemoteBodyRequest<Header>) -> Self::RemoteBodyResult {
+			err("Not implemented on test node".into())
```

Comment:
This should be refactored into a utility function.

Reply:
Are you implying that all `test helpers` should be utility functions?

Otherwise please explain!

Reply:
(Same person, just different account)

I meant that we could have a

```rust
fn<T> not_implemented_in_tests() -> Result<T, Error> {
	err("Not implemented on test node".into())
}
```